### ISSUE TITLE
[Feature] - Dual OPN support

### ISSUE THREAD

**Issue by denjhang** [BambooTracker/BambooTracker#431]
Recently I made a 2xOPN hardware VGM player module.I want to use BambooTracker to write music for it.

**Comment by rerrahkr** [comment]
I apologize for not responding to your report for so long.
Support for multiple OPNAs is out of scope in our current plan, but several people have requested it, so we may work on it if it seems easy to implement.

**Comment by denjhang** [comment]
I actually asked for dual OPNs, not dual OPNAs, similar to the ZX Spectrum Turbo Sound.
Of course I would also like to have double OPNA. I wish BambooTracker would support more chips.

**Comment by NEO-SPECTRUMAN** [comment]
yes there is no trackers that support ZX Spectrum's TSFM (2203 x2)
source code of tfmmaker is lost
and it support only FM part of chip...
